doc/manifest-tool-guide.md

```diff
@@ -28,10 +28,9 @@ Sample output:
   paamfhAn5m1NM4UUu5mmvVaHQy8Fb65bkpbaNrvKwX3YMKdjzi2
 ```
 
-The first value is the master public key. Add the public key to the config
-for this validator. A one-word comment must be added after the key (for example
-*ThisServersName*). Any other rippled trusting the validator needs to add the
-master public key to its config. Only add keys received from trusted sources.
+The first value is the master public key. Any other rippled trusting the
+validator needs to add the master public key to its config. Only add keys
+received from trusted sources.
 
 The second value is the corresponding master secret key. **DO NOT INSTALL THIS
```

Comment:
I'd change it if it didn't affect the python tests.
I'm hoping to have the ripple-libpp based tool ready soon, which will remove the need to copy and paste these keys anywhere.